force-app/main/default/classes/SOQL.cls

```diff
@@ -192,7 +240,7 @@ public with sharing class SOQL {
      * @return A list of Contacts
      */
     public static List<Contact> getBirthdateContacts() {
-        return null; // Replace null with the variable you used to store the result
+        return [SELECT Birthdate FROM Contact WHERE Birthdate != null ORDER BY Birthdate DESC LIMIT 3];
```

Comment:
in queries, `NULL` is a capitalized keyword, unlike in apex where it is all lowercase